observability/otel/otel-collector/otel-collector-processors.adoc

```diff
@@ -574,6 +576,342 @@ config:
+
+The Tail Sampling Processor samples traces according to user-defined policies when all of the spans are completed. Tail-based sampling enables you to filter the traces of interest and reduce your data ingestion and storage costs.
+
+:FeatureName: The Tail Sampling Processor
+include::snippets/technology-preview.adoc[]
+
+This processor reassembles spans into new batches and strips spans of their original context.
+
+[TIP]
+====
+* In pipelines, place this processor downstream of any processors that rely on context: for example, after the Kubernetes Attributes Processor.
+* If scaling the Collector, ensure that one Collector instance receives all spans of the same trace so that this processor makes correct sampling decisions based on the specified sampling policies. You can achieve this by setting up two layers of Collectors: the first layer of Collectors with the Load Balancing Exporter, and the second layer of Collectors with the Tail Sampling Processor.
+====
+
+.Example of the OpenTelemetry Collector custom resource when using the Tail Sampling Processor
+[source,yaml]
+----
+# ...
+config:
+  processors:
+    tail_sampling: # <1>
+      decision_wait: 30s # <2>
+      num_traces: 50000 # <3>
+      expected_new_traces_per_sec: 10 # <4>
+      policies: # <5>
+        [
+          {
+            <definition_of_policy_1>
+          },
+          {
+            <definition_of_policy_2>
+          },
+          {
+            <definition_of_policy_3>
+          },
+        ]
+# ...
+----
+<1> Processor name.
+<2> Optional: Decision delay time, counted from the time of the first span, before the processor makes a sampling decision on each trace. Defaults to `30s`.
+<3> Optional: The number of traces kept in memory. Defaults to `50000`.
+<4> Optional: The expected number of new traces, which is helpful for allocating data structures. Defaults to `0`.
+<5> Definitions of the policies for trace evaluation. The processor evaluates each trace against all of the specified policies and then either samples or drops the trace.
+
+You can choose and combine policies from the following list:
+
+* The following policy samples all traces:
++
+----
+# ...
+      policies:
+        [
+          {
+            name: <always_sample_policy>,
+            type: always_sample,
+          },
+        ]
+# ...
+----
+
+* The following policy samples only traces of a duration that is within a specified range:
++
+----
+# ...
+      policies:
+        [
+          {
+            name: <latency_policy>,
+            type: latency,
+            latency: {threshold_ms: 5000, upper_threshold_ms: 10000} # <1>
+          },
+        ]
+# ...
+----
+<1> The provided `5000` and `10000` values are examples. You can roughly estimate the desired latency values by looking at the earliest start time value and latest end time value. If you omit the `upper_threshold_ms` field, this policy samples all latencies greater than the specified `threshold_ms` value.
```

Comment:
```suggestion
<1> The provided `5000` and `10000` values are examples. You can estimate the desired latency values by looking at the earliest start time value and latest end time value. If you omit the `upper_threshold_ms` field, this policy samples all latencies greater than the specified `threshold_ms` value.
```

Reply:
Thank you for the feedback. Applying your suggestion.